backend/database/init/01_db_structure.sql

```diff
@@ -0,0 +1,24 @@
+-- Create departments table
+CREATE TABLE IF NOT EXISTS departments (
```

Comment:
This can be changed to do

IF EXISTS (SELECT 1 FROM information_schema.tables WHERE table_name = 'departments' AND table_schema = 'public') THEN
    -- If it exists, delete the table
    DROP TABLE departments;
END IF;